scripts/install-worker.sh

```diff
@@ -30,6 +30,7 @@ validate_env_set CNI_PLUGIN_VERSION
 validate_env_set KUBERNETES_VERSION
 validate_env_set KUBERNETES_BUILD_DATE
 validate_env_set PULL_CNI_FROM_GITHUB
+validate_env_set SONOBUOY_E2E_REGISTRY
```

Comment:
To keep the expected behavior (skip the sonobuoy block below if unset), we can initialize the variable rather than assert that it's set.

```
SONOBUOY_E2E_REGISTRY="${SONOBUOY_E2E_REGISTRY:-}"
```

IMO, that's probably how we should handle that. Good catch and thanks for the contribution!